util/types/datum.go

```diff
@@ -1408,8 +1423,8 @@ func (d *Datum) ToFloat64(sc *variable.StatementContext) (float64, error) {
 		f, err := d.GetMysqlTime().ToNumber().ToFloat64()
 		return f, err
 	case KindMysqlDuration:
-		f, _ := d.GetMysqlDuration().ToNumber().ToFloat64()
-		return f, nil
+		f, err := d.GetMysqlDuration().ToNumber().ToFloat64()
+		return f, err
```

Comment:
Should we trace this err?

Reply:
I didn't think too much, maybe the traces are intentionally avoided, so just keep it the same as other cases.

Maybe we can do it in another PR to add missing traces.